docs/sources/static/release-notes.md

```diff
@@ -28,6 +28,16 @@ For a complete list of changes to Grafana Agent, with links to pull requests and
 [release-notes-flow]: "/docs/grafana-cloud/ -> ../flow/release-notes"
 {{% /docs/reference %}}
 
+## v0.36
+
+### Breaking change: The prometheus default value of `retry_on_http_429` is overriden to `true` for the `queue_config` in `remote_write`
```

Comment:
Why "prometheus default value"?

Reply:
because the default value is taken from prometheus (https://prometheus.io/docs/prometheus/2.42/configuration/configuration/#remote_write). This is the page which is referred to in the static documentation of the agent. This might actually be confusing for the users that we refer to this page which clearly says that the retry default value is false but in the agent it will be actually true

Reply:
I see. I find the phrasing confusing though. I think we should just say:

> Breaking change: The default value of `retry_on_http_429` is overriden to `true` for the `queue_config` in `remote_write` in `metrics` config.

And then add a note saying something like:
> NOTE: the default set by Grafana Agent Static Mode is different to the default set by Prometheus.

Finally, shouldn't this be mentioned somewhere in the static mode docs? What do you think?

Reply:
I extended the note in the static doc